src/mscorlib/model.xml

```diff
@@ -1924,7 +1983,18 @@
       <Member Name="get_OSVersion" />
       <Member Name="get_ProcessorCount" />
       <Member Name="get_StackTrace" />
+      <Member Name="get_CurrentDirectory" />
+      <Member Name="get_Is64BitOperatingSystem" />
+      <Member Name="get_Is64BitProcess" />
+      <Member Name="get_SystemPageSize" />
+      <Member Name="get_UserDomainName" />
+      <Member Name="get_UserInteractive" />
+      <Member Name="get_UserName" />
+      <Member Name="get_WorkingSet" />
       <Member Name="GetEnvironmentVariable(System.String)" />
+      <Member Name="GetFolderPath(System.Environment+SpecialFolder)" />
+      <Member Name="GetFolderPath(System.Environment+SpecialFolder,System.Environment+SpecialFolderOption)" />
+      <Member Name="GetLogicalDrives" />
```

Comment:
We're using System.Environment that's defined in corefx; we shouldn't need to export anything additional for it from coreclr.